jib-core/src/main/java/com/google/cloud/tools/jib/api/Containerizer.java

```diff
@@ -28,6 +32,14 @@
 // TODO: Add tests once JibContainerBuilder#containerize() is added.
 public class Containerizer {
 
+  /**
+   * The default directory for caching the base image layers, in {@code [user cache
+   * home]/google-cloud-tools-java/jib}.
+   */
+  // TODO: Reduce scope once plugins are migrated to use the new Jib Core API.
```

Comment:
Seems an odd comment when this is a newly-introduced field?

Reply:
This is currently used to set the default in `PluginsConfigurationProcessor`. However, once the plugins are migrated to use the new API (and this class), it won't need to use this field to set defaults as this class sets the default itself.